scripts/ec2-stage

```diff
@@ -25,10 +25,19 @@ SOURCE_BUCKET=""
 KEY_NAME=""
 SECURITY_GROUP=""
 INSTANCE_PROFILE=""
-
+SUPPORTED_OSES=("linux" "windows")
+TARGET_OS_LIST=${SUPPORTED_OSES[@]}
 
 source $(dirname "${0}")/publishing-common.sh
 
+supported_os() {
+	local os=$1
+	for entry in ${SUPPORTED_OSES[@]} ; do
+		test "$os" == "$entry" && return $(true)
```

Comment:
IMO the addition of punctuation where it's not necessary detracts from the readability of the code. The [Google bash styleguide](https://google.github.io/styleguide/shell.xml#Variable_expansion) describes this topic as "too controversial for a mandatory regulation", so I suspect that we'll see different styles based on personal preference for as long as we're writing shell. However, the argument for consistency is valid, so I'm OK with the change.